prompting/base/prompting_miner.py

```diff
@@ -23,7 +23,7 @@
 from prompting.protocol import StreamPromptingSynapse
 from prompting.base.miner import BaseStreamMinerNeuron
 from datetime import datetime
-
+from typing import List, Dict
```

Comment:
python now has built-in types which are generally preferred over typing

Reply:
Yeah, this is some old code from python 3.8.
It's already on main tho, this PR is just to keep pre-staging up to date.